core/mbed/uvisor-lib/box_config.h

```diff
@@ -48,35 +53,35 @@ UVISOR_EXTERN const uint32_t __uvisor_mode;
 /* this macro selects an overloaded macro (variable number of arguments) */
 #define __UVISOR_BOX_MACRO(_1, _2, _3, _4, NAME, ...) NAME
 
-#define __UVISOR_BOX_CONFIG(box_name, acl_list, acl_list_count, stack_size, context_size) \
```

Comment:
This was changed so that we would avoid confusion (cognitive overhead) between the box name as provided by UVISOR_BOX_NAME and the prefix we stick on the front of variables declared or defined by the macro.